superset-frontend/src/dashboard/components/CssEditor.jsx

```diff
@@ -19,12 +19,9 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import Select from 'src/components/Select';
-import AceEditor from 'react-ace';
-import 'brace/mode/css';
-import 'brace/theme/github';
 import { t } from '@superset-ui/core';
-
-import ModalTrigger from '../../components/ModalTrigger';
+import ModalTrigger from 'src/components/ModalTrigger';
+import { CssEditor as AceCssEditor } from 'src/components/AsyncAceEditor';
```

Comment:
Why not just import it as CssEditor? For this or other editors, the Ace branding seems relatively unimportant (and who knows, maybe something we'll pivot away from one day).

Reply:
Because this file itself is named `CssEditor`. Thought it'd be beneficial to differentiate.